src/types/reduxTypes/eventsStateTypes.ts

```diff
@@ -24,3 +24,10 @@ export interface IEventsListProps {
 export interface IEventsListParams {
   page: number
 }
+
+export interface ICreateEventParams {
+  header: string
+  paragraph: string
+  imageDescription: string
+  imageURL: File | null
```

Comment:
should imageURL be file or string?

Reply:
image must be or File or null for create